tests/typecheck/test_test.yml

```diff
@@ -26,3 +26,19 @@
             reveal_type(self.client) # N: Revealed type is "django.test.client.Client"
             response = self.client.get('/', format="json")
             self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+
+- case: test_api_client_get
+  main: |
+    from rest_framework import test
+
+    client = test.APIClient()
+    response = client.get('/', format="json")
+    reveal_type(response) # N: Revealed type is "rest_framework.response._MonkeyPatchedResponse"
+
+- case: test_api_client_factory_get
+  main: |
+    from rest_framework import test
+
+    factory = test.APIRequestFactory()
+    request = factory.get("/")
+    reveal_type(request) # N: Revealed type is "rest_framework.request.Request"
```

Comment:
Sorry, we don't ordinarily add tests for type hints, only for complex cases like decorators and overloads. Otherwise they would slow down the tests too much

Reply:
Should I revert changes to this file?

Reply:
Yes please

Reply:
Reverted ✔️